### PR TITLE
Updates content on home page, menus, and scanner landing page

### PR DESCRIPTION
**Pull request by hxlnt**
Changes include:

- Updates menus to combine "Try" and "Install sections" into one called "Get webhint"
- Updates copy on scanner landing page as part of scanner retirement
- Updates copy on home page and changes absolute links to relative links
- Adds some needed styles for new content

### CODE REVIEW

**Comment by hxlnt** [comment]
Note: I'll add a follow-up PR in the `hint` repo to update some of the docs in the User Guide.

File: src/webhint-theme/layout/partials/home.ejs [hunk 1]
```diff
@@ -18,18 +18,17 @@
...
                         <h3 class="subtitle subtitle--home">Develop</h3>
-                        <p class="feature__highlight">Use the <a href="https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint">VS Code extension</a> to get real-time feedback in your IDE when writing code</p>
+                        <p class="feature__highlight">Use the <a href="/docs/user-guide/extensions/vscode-webhint/">VS Code extension</a> to get real-time feedback in your IDE when writing code.</p>
                     </div>
                     <div class="module feature">
                         <img src="/images/webhint_firefox.svg" alt="" role="presentation" />
                         <h3 class="subtitle subtitle--home">Debug</h3>
-                        <p class="feature__highlight">The <a href="https://webhint.io/docs/user-guide/extensions/extension-browser/">browser extension</a> lets you scan a site from within your browser's DevTools.</p>
-
+                        <p class="feature__highlight">Test your site from the browser with the <a href="/docs/user-guide/extensions/extension-browser/">DevTools extension</a>. Using Edge? webhint now ships with Edge DevTools!</p>
```
Comment (antross) [inline src/webhint-theme/layout/partials/home.ejs:26]: Should we link to the Edge Issues panel documentation here as well for users to learn more?
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/issues/

**Review by antross — APPROVED** [review]
One question, but overall looks good and I like the updates. Thanks for putting them together!

File: src/webhint-theme/layout/partials/home.ejs (hunk 1 above)
Comment (antross) [reply src/webhint-theme/layout/partials/home.ejs:26]: I'll leave this for once we integrate the content updates from the hint repo.

**Pull request merged**